index.html

```diff
@@ -0,0 +1,164 @@
+                     <h5>Mike Nigeria</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>In my opinion, women are tech are a big asset to the selves and the community. So I will encourage every girl to get into the tech industry and explore options.</p>
+                     </div>
+                 </div>
+                 <div>
+                     <!-- <button id="dropbtn">MORE <img class="dropimg" src="images/dropdown.png"></button> -->
+                    </div>
+                    <div class="hide"></div>
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Steph Bahamas</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>Tech is the leading industry in the world. It comes with magically powers and well cherished when women invade the tech space.</p>
+                     </div>
+                 </div>
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Steph Bahamas</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>The sole idea of advocating for black girls in tech is to bring value to every black girl and let them see the potential in them.</p>
+                     </div>
+                 </div>
+                 <div class="speakers">
+                    <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                    <div class="column">
+                    <h5>Jane Cameroon</h5>
+                    <h4>Guest</h4>
+                    <hr>
+                    <p>The version, the goal, the technologies, the growth, every girl child should be a part of it. Lets make tech a comfort place for every child</p>
+                    </div>
+                </div>
+                <div class="speakers">
+                    <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                    <div>
+                    <h5>Precillia Cameroon</h5>
+                    <h4>Guest</h4>
+                    <hr>
+                    <p>The version, the goal, the technologies, the growth, every girl child should be a part of it. Lets make tech a comfort place for every child</p>
+                    </div>
+                </div>
+                    </div>
+            </section>
+            <footer class="last--section">
+                <div class="footer">
+                    <p>partner</p>
+                    <hr>
+                    <div class="grid-container">
+                      <h3>yogaya Inc</h3>
+                      <h2>DataGend</h2>
+                    </div>
+                </div>
+            </footer>
```

Comment:
- [ ] Please implement the footer part of this website according to the behance design below 👇🏻 

![image](https://user-images.githubusercontent.com/4247740/148642474-0b3a8678-f8ec-4a6e-84b0-7fe50db05280.png)

Reply:
Resolved.

The footer has been part implemented according to the Behance design.